frontend/src/components/pages/signin/login-form.tsx

```diff
@@ -44,16 +49,21 @@ function LoginForm() {
           name="email"
           render={({ field }) => (
             <FormItem>
-              <FormLabel className="text-foreground">Email</FormLabel>
+              <FormLabel htmlFor="email-input" className="text-foreground">
+                Email
+              </FormLabel>
               <FormControl>
                 <Input
                   className="text-foreground"
                   type="email"
+                  id="email-input"
                   placeholder="marcelo@example.com"
+                  aria-invalid={!!form.formState.errors.email}
+                  aria-errormessage="email-error"
                   {...field}
                 />
               </FormControl>
-              <FormMessage />
+              <FormMessage id="email-error" />
```

Comment:
# Comentario 

Nao adicionar Ids ao JSX apenas para serem referenciados para testes. 

![image](https://github.com/talDoFlemis/brain.test/assets/74384665/b6319130-acb7-42a2-8e8d-04f6dae6bbc1)

[Fonte](https://www.freecodecamp.org/news/write-unit-tests-using-react-testing-library/)

Reply:
# Ideia 
Adicionar um "aria-role" no componente do ShadCn

Reply:
> # Comentario 
> 
> Nao adicionar Ids ao JSX apenas para serem referenciados para testes. 
> 
> ![image](https://github.com/talDoFlemis/brain.test/assets/74384665/b6319130-acb7-42a2-8e8d-04f6dae6bbc1)
> 
> [Fonte](https://www.freecodecamp.org/news/write-unit-tests-using-react-testing-library/)

Exatamente, por isso estou usando o id padrão e nos testes sempre to buscando pela role ou label pq seria a forma acessível l